### PR TITLE
Cortex XDR Script Execution playbooks

### PR DESCRIPTION
**Pull request by EliorKedar**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Script Execution playbooks, Including the following:

- Cortex XDR - Run Script Delete file
- Cortex XDR - Run Script Execute commands
- Cortex XDR - Run Script file exists
- Cortex XDR - Run Script Kill process
- Cortex XDR - Run Script
- Cortex XDR - Run Snippet code script

## Screenshots
Cortex_XDR_-_Run_script_delete_file
Cortex_XDR_-_Run_script_execute_commands
Cortex_XDR_-_Run_script_file_exists
Cortex_XDR_-_Run_script_kill_process
Cortex_XDR_-_Run_script
Cortex_XDR_-_Run_snippet_code_script

## Minimum version of Demisto
- [ ] 5.0.0
- [x] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

**Comment by EliorKedar** [comment]
@michalgold  I've updated the PR with wrapping retrieve file playbook inputs to support generic file path.
waiting for approval.

**Comment by ShirleyDenkberg** [comment]
@michalgold Doc review completed.

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Run_script.yml
Comment (altmannyarden) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Run_script.yml:16]: Add the command name the users can run to retrieve the script parameters

**Comment by altmannyarden** [comment]
You also can add checks to see if the inputs exists

**Review by altmannyarden — APPROVED** [review]

**Pull request merged**
